### PR TITLE
replaced bidcrypto dep with fxa-jwtool

### PR DESCRIPTION
**Pull request by dannycoates**
just a dev dep, but now ought to work with newer node versions...

### CODE REVIEW

**Comment by GitCop** [comment]
Thanks for contributing! Unfortunately, I'm here to tell you there were the following style issues with your Pull Request:

 * Commit: d5233c45694a9e7e2acbe385ca137fc149492925
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-oauth-server

---
This message was auto-generated by https://gitcop.com

**Comment by rfk** [comment]
thanks @dannycoates - LGTM, but GitCop and travis say otherwise...

File: test/api.js [hunk 1]
```diff
@@ -4,9 +4,10 @@
 
 const url = require('url');
 const assert = require('insist');
-const bidcrypto = require('browserid-crypto');
 const nock = require('nock');
 const buf = require('buf').hex;
+const generateRSAKeypair = require('generate-rsa-keypair');
```
Comment (rfk) [inline test/api.js]: How confident are we in this library and its ability to do the right thing w.r.t all the terrible things that can go wrong when generating keys?  It looks like a pretty thin wrapper over SSL but I haven't reviewed it thoroughly.

Comment (dannycoates) [inline test/api.js]: i don't anticipate us generating prod keys with it so i for one don't care all that much, but yeah it just a wrapper around the one openssl function.

Comment (rfk) [inline test/api.js]: > i don't anticipate us generating prod keys with it

Agreed, but these things can easily propagate to unintended uses once people notice them

Comment (dannycoates) [inline test/api.js]: yeah, good that it only does one thing then :)

**Comment by rfk** [comment]
r+; but not merging this until after the train cut

**Comment by rfk** [comment]
@dannycoates please rebase and merge at your convenience

**Comment by GitCop** [comment]
Thanks for contributing! Unfortunately, I'm here to tell you there were the following style issues with your Pull Request:

 * Commit: a2fa2df547d89776b475a3d6b0fb5686ddd2fba9
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-oauth-server

---
This message was auto-generated by https://gitcop.com

**Pull request merged**